app/controllers/patients_controller.rb

```diff
@@ -28,12 +28,13 @@ def show
                                      Hash[current_user.jurisdiction.subtree.pluck(:id, :path).map { |id, path| [id, path] }]
                                    end
 
-    # Group members if this is HOH
-    @group_members = @patient.dependents_exclude_self.where(purged: false)
+    # Household members (dependents) for the HOH excluding HOH
+    @dependents_exclude_hoh = @patient.dependents_exclude_self.where(purged: false)
 
-    # All group members regardless if this is not HOH
+    # All household members regardless if this is not HOH
     dependents = current_user.get_patient(@patient.responder_id)&.dependents
-    @all_group_members = ([@patient] + (dependents.nil? ? [] : dependents)).uniq
+    @household_members = ([@patient] + (dependents.nil? ? [] : dependents)).uniq
+    @household_members_with_ce_in_exposure = dependents.nil? ? [] : dependents.where(isolation: false, continuous_exposure: true)
```

Comment:
this is used for the LDE update in the monitoring status modal which does not include the current monitoree, so that's not needed. if it's unclear we can rename it

Reply:
Ah okay, yeah maybe call it `dependents_with_ce_in_exposure`.

Reply:
The thing with this is that it's not HoH dependent, which is what I've been using "dependents" for rather than household members.  However I can see the proceeding line is now confusing in that regard.  I will swap to that variable name

Reply:
What do you mean "not HoH dependent"?

Reply:
`@household_members` gets all the members of the household including the current monitoree, regardless of if they are the HoH or not whereas `@dependents_exclude_hoh` ONLY works on the HoH and will return an empty array if you run it on a dependent

Reply:
Right, so I guess I'm confused why you're saying that that is inconsistent with changing this variable to `dependents_with_ce_in_exposure` since this will be empty if the person is not a HoH.

Reply:
Okay I see the changes you made here: https://github.com/SaraAlert/SaraAlert/pull/450/commits/720a0e06072fbd86e67daa1f1087e4aefdba7217

Looks like there's still two instances of the `dependents` variable there that need to be updated to be `household`. I do think it's really unclear at first why `patient` is not being added to the `household_members_with_ce_in_exposure`, so I'd at least add a comment explaining that above that line.

Reply:
`@household_members_with_ce_in_exposure` is _**only**_ used for that update LDE section as a part of the monitoring status modal, which as you know has a very specific use case.  I can add an `excludes_patient` to the end or something but then the variable name is getting way too long. Although it is only used in that one place so I will added for clarity.  Also will add a comment

Reply:
all set